index.bs

```diff
@@ -591,6 +591,12 @@ Follow the <a href="https://www.w3.org/2001/tag/doc/promises-guide#one-time-even
 in the <strong><a href="https://www.w3.org/2001/tag/doc/promises-guide">Writing
 Promise-Using Specifications</a></strong> guideline.
 
+<h3 id="promises-and-events">Events should fire before promises resolve</h3>
+
+In the case that an event is being dispatches as part of a promise being
```

Comment:
```suggestion
In the case that an event is being dispatched as part of a promise being
```

Reply:
I don't know why I always make that typo!